descriptors/ModuleDescriptor-template.json

```diff
@@ -54,7 +54,8 @@
       "displayName": "RTAC - all permissions",
       "description": "Entire set of permissions needed to use RTAC",
       "subPermissions": [
-        "rtac.batch.post"
+        "rtac.batch.post",
+        "rtac.holdings.item.get"
```

Comment:
We also need the actual `rtac.holdings.item.get` permission to be created as a permission set, similar to `rtac.batch.post` on line 48.

Reply:
done.